spring-cloud-gcp-vision/src/main/java/org/springframework/cloud/gcp/vision/CloudVisionTemplate.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Copyright 2018 original author or authors.
```

Comment:
We need to have a fist year as well: the date a project has been started.

```suggestion
 *  Copyright 2017-2018 original author or authors.
```